### PR TITLE
Trigger build workflows for `model` on sqoop

### PR DESCRIPTION
**Pull request by dfsnow**
This PR adds triggers to rebuild the main `model` input views/tables on sqoop. Once https://github.com/ccao-data/data-architecture/pull/255 is merged, this will rebuild the full table every morning once sqoop is finished running.

### CODE REVIEW

File: scripts/dispatch-dbt-workflow.sh [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
+    "https://api.github.com/repos/ccao-data/data-architecture"
+)
+GH_API_WORKFLOWS=(
+    "test_dbt_models.yaml"
+    "build_and_test_dbt.yaml"
+)
+GH_API_WORKFLOW_INPUTS=(
+    ""
+    "model.vw_card_res_input model.vw_pin_condo_input"
+)
```
Comment (dfsnow) [inline scripts/dispatch-dbt-workflow.sh:18]: This is pretty hideous but works fine. Just iterates through the index of the arrays.

Comment (jeancochrane) [reply scripts/dispatch-dbt-workflow.sh:18]: **[Nitpick, non-blocking]** I love representing data in shell scripts 🙈 Perhaps a brief comment explaining the data structure would help with readability?

File: scripts/dispatch-dbt-workflow.sh [hunk 2]
```diff
@@ -28,12 +38,22 @@ GH_TOKEN=$(curl -s -L \
...
-    "$GH_API_REPO"/actions/workflows/"$GH_DBT_WORKFLOW"/dispatches \
-    -d '{"ref": "master"}'
+# Use the token to call the API and dispatch the workflows
+echo "Dispatching workflows"
+for i in ${!GH_API_REPOS[*]}; do
+    GH_API_PARAMS=$(
+        if [ -z "${GH_API_WORKFLOW_INPUTS[$i]}" ]; then
+            echo '{"ref": "master"}'
+        else
+            echo "{\"ref\": \"master\", \"inputs\": {\"models\": \"${GH_API_WORKFLOW_INPUTS[$i]}\"}"
```
Comment (jeancochrane) [inline scripts/dispatch-dbt-workflow.sh]: **[Nitpick]** Hm, I might be reading wrong here but I think we may be missing a closing brace:

```suggestion
            echo "{\"ref\": \"master\", \"inputs\": {\"models\": \"${GH_API_WORKFLOW_INPUTS[$i]}\"}}"
```

**Review by jeancochrane — COMMENTED** [review]
Looking great, just one small nitpick that I think may be a syntax error!

File: scripts/dispatch-dbt-workflow.sh (hunk 1 above)
Comment (dfsnow) [reply scripts/dispatch-dbt-workflow.sh:18]: Done in 3bee4a2!

File: scripts/dispatch-dbt-workflow.sh (hunk 2 above)
Comment (dfsnow) [reply scripts/dispatch-dbt-workflow.sh]: Nice catch. Fixed in 3bee4a2!

**Review by jeancochrane — APPROVED** [review]
Good to go!

**Pull request merged**